README.md

```diff
@@ -344,6 +344,15 @@ Or if you have certificate in `PFX` or `PKCS12` format, setting code can be like
 ### Azure Active Directory(Azure AD)
 Azure AD is Microsoft’s multi-tenant, cloud-based directory and identity management service, you can refer to the [System Variables](#system-variables) section for more details.
 
+### AWS Signature v4
+AWS Signature version 4 is used to authenticate requests to AWS services. For more information, see [AWS documentation](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html). `sessionToken` as well as the `X-AWS-Region` and `X-AWS-Service` headers are optional.
+```http
+GET https://httpbin.org/aws-auth HTTP/1.1
+X-Authentication-Type: AWS accessId accessKey sessionToken
+X-AWS-Region: awsRegion
+X-AWS-Service: awsService
```

Comment:
@sebastian-fredriksson-bernholtz I prefer the **AWS Signature v4** auth mechanism built on the `Authorization` header as other Authorization schemas like **Basic Auth**, **Digest Auth**, and **Azure AD**. We don't need to invent these three new headers `X-Authentication-Type`, `X-AWS-Region`, and `X-AWS-Service`.

How about coming up with a syntax for `AWS Signature v4` auth scheme like the following:
```http
Authorization: AWS accessKeyId secretAccessKey [token:sessionToken] [region:regionName] [service:serviceName]
```
And the `token`, `region` and `service` parameters are optional.

Reply:
The reason I used a new header was that I didn't want it confused with the actual HTTP Authorization Header that will generated and sent. But I realise now that it shouldn't be a concern as the Digest authentication is already functioning like this.